### DISCUSSION TITLE
Any docs or samples for Shiny.MSAL

### DISCUSSION THREAD

**Discussion by henda79** [shinyorg/shiny#1032] [Q&A] [answered]
Hi Guys, I use MSAL in my project and when I was browsing through NuGet I found the Shiny.MSAL library, however I can't find any docs or samples, or even the source code for it here.

Any pointers would be great and code sample would even better :)

**Comment by aritchie** [comment, marked as answer]
Nope.  This isn't something I'm supporting externally and was already brought back to private source.
